src/main/java/teammates/storage/entity/Student.java

```diff
@@ -147,10 +147,11 @@ public Long getRegistrationKey() {
         return registrationKey;
     }
 
+    // TODO method unused
```

Comment:
I think you remove this method. Registration key is set by the database for students